src/pages/dashboard/components/Profile/control/index.tsx

```diff
@@ -1,14 +1,19 @@
+import { DISCORD_OAUTH_URL } from "~/common/constants";
 import { InfoRow } from "~/components/common/Info";
 import { Price } from "~/components/common/Price";
 import { Text } from "~/components/common/Text";
+import { Button } from "~/components/controls/Button";
 import { BlurBackground } from "~/components/layout/BlurBackground";
 import { Flex } from "~/components/layout/Flex";
 import { usePlayerStore } from "~/stores/usePlayerStore";
 import { shortenAddress } from "~/utils/shortenAddress";
 import { CreatePlayerBanner } from "../CreatePlayerBanner";
+import Image from "next/image";
+import Link from "next/link";
 
 export const Profile = () => {
```

Comment:
Here we are editing the `control` Profile component, but also the `variant` should be changed